Makefile

```diff
@@ -232,6 +232,14 @@ test-java: build-test-java
 	echo "Running tests against $(LIB_PROFILER)"
 	$(JAVA) "-Djava.library.path=$(TEST_LIB_DIR)" $(TEST_FLAGS) -ea -cp "build/test.jar:build/jar/*:build/lib/*" one.profiler.test.Runner $(TESTS)
 
+test-java-from-release:
+	mkdir -p build/jar
+	cp $(JARS_DIRECTORY)/* build/jar
+	$(MAKE) build/$(TEST_JAR)
+	cp -r $(RELEASE_DIRECTORY)/bin build
+	cp -r $(RELEASE_DIRECTORY)/lib build
+	$(JAVA) "-Djava.library.path=$(TEST_LIB_DIR)" $(TEST_FLAGS) -ea -cp "build/test.jar:build/jar/*:build/lib/*" one.profiler.test.Runner $(TESTS)
```

Comment:
Someone needs to set `RELEASE_DIRECTORY` and `JARS_DIRECTORY` externally, and this is mainly for the purpose of CI in GHA. So why should it be a Makefile target? Can GHA workflow do the required preparations and then just call `make test` as usual?